$MirOS: ports/print/cups/patches/patch-scheduler_Makefile,v 1.1.7.1 2005/03/18 15:51:00 tg Exp $
--- scheduler/Makefile.orig	Sun Mar  5 16:47:49 2006
+++ scheduler/Makefile	Mon Jun 19 20:06:13 2006
@@ -127,22 +127,6 @@ install:	all
 	echo Creating $(SERVERROOT)/ppd...
 	$(INSTALL_DIR) -m 755 $(SERVERROOT)/ppd
 	-chgrp $(CUPS_GROUP) $(SERVERROOT)/ppd
-	echo Creating $(STATEDIR)...
-	$(INSTALL_DIR) -m 755 $(STATEDIR)
-	echo Creating $(STATEDIR)/certs...
-	$(INSTALL_DIR) -m 511 $(STATEDIR)/certs
-	-chgrp $(CUPS_PRIMARY_SYSTEM_GROUP) $(STATEDIR)/certs
-	echo Creating $(LOGDIR)...
-	$(INSTALL_DIR) -m 755 $(LOGDIR)
-	echo Creating $(REQUESTS)...
-	$(INSTALL_DIR) -m 710 $(REQUESTS)
-	-chgrp $(CUPS_GROUP) $(REQUESTS)
-	echo Creating $(REQUESTS)/tmp...
-	$(INSTALL_DIR) -m 1770 $(REQUESTS)/tmp
-	-chgrp $(CUPS_GROUP) $(REQUESTS)/tmp
-	echo Creating $(CACHEDIR)...
-	$(INSTALL_DIR) -m 775 $(CACHEDIR)
-	-chgrp $(CUPS_GROUP) $(CACHEDIR)
 #	echo Creating $(CACHEDIR)/ppd...
 #	$(INSTALL_DIR) -m 755 $(CACHEDIR)/ppd
 #	-chgrp $(CUPS_GROUP) $(CACHEDIR)/ppd
@@ -179,13 +163,13 @@ uninstall:	all
 
 cupsd:	$(CUPSDOBJS) libmime.a ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cupsd $(CUPSDOBJS) libmime.a \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cupsd $(CUPSDOBJS) libmime.a \
 		$(LIBZ) $(SSLLIBS) $(LIBSLP) $(LIBLDAP) $(PAMLIBS) \
 		$(LIBPAPER) $(LIBMALLOC) $(CUPSDLIBS) $(LIBS)
 
 cupsd-static:	$(CUPSDOBJS) libmime.a ../cups/libcups.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cupsd-static $(CUPSDOBJS) libmime.a \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cupsd-static $(CUPSDOBJS) libmime.a \
 		$(LIBZ) $(SSLLIBS) $(LIBSLP) $(LIBLDAP) $(PAMLIBS) \
 		../cups/libcups.a $(COMMONLIBS) $(LIBZ) $(LIBPAPER) \
 		$(LIBMALLOC) $(CUPSDLIBS)
@@ -197,7 +181,7 @@ cupsd-static:	$(CUPSDOBJS) libmime.a ../
 
 cups-deviced:	cups-deviced.o util.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cups-deviced cups-deviced.o util.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cups-deviced cups-deviced.o util.o $(LIBS)
 
 
 #
@@ -206,7 +190,7 @@ cups-deviced:	cups-deviced.o util.o ../c
 
 cups-driverd:	cups-driverd.o util.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cups-driverd cups-driverd.o util.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cups-driverd cups-driverd.o util.o $(LIBS)
 
 
 #
@@ -215,7 +199,7 @@ cups-driverd:	cups-driverd.o util.o ../c
 
 cups-lpd:	cups-lpd.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cups-lpd cups-lpd.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cups-lpd cups-lpd.o $(LIBS)
 
 
 #
@@ -224,7 +208,7 @@ cups-lpd:	cups-lpd.o ../cups/$(LIBCUPS)
 
 cups-polld:	cups-polld.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o cups-polld cups-polld.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o cups-polld cups-polld.o $(LIBS)
 
 
 #
@@ -244,7 +228,7 @@ libmime.a:	$(MIMEOBJS)
 
 testdirsvc:	testdirsvc.o
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o testdirsvc testdirsvc.o $(COMMONLIBS) $(NETLIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o testdirsvc testdirsvc.o $(COMMONLIBS) $(NETLIBS)
 
 
 #
@@ -253,7 +237,7 @@ testdirsvc:	testdirsvc.o
 
 testmime:	testmime.o libmime.a ../cups/libcups.a
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o $@ testmime.o libmime.a ../cups/libcups.a \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o $@ testmime.o libmime.a ../cups/libcups.a \
 		$(COMMONLIBS) $(LIBZ) $(SSLLIBS)
 
 
@@ -263,7 +247,7 @@ testmime:	testmime.o libmime.a ../cups/l
 
 testspeed:	testspeed.o ../cups/$(LIBCUPS)
 	echo Linking $@...
-	$(CC) $(LDFLAGS) -o testspeed testspeed.o $(LIBS)
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o testspeed testspeed.o $(LIBS)
 
 
 #
